src/test/resources/application.properties

```diff
@@ -3,3 +3,6 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
+
+jwt.expiration= 10000
+jwt.secret=secretmessagemfdkfgdpwe32fosj08sjsnflwjtphsowh3ip28rnbsfk285twsdfjnweo
```

Comment:
Please don`t push sensitive information

Reply:
let's provide this value from env properties